sim/paladin/aura_mastery.go

```diff
@@ -0,0 +1,56 @@
+package paladin
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+	"github.com/wowsims/sod/sim/core/stats"
+)
+
+func (paladin *Paladin) registerAuraMastery() {
+
+	if !paladin.hasRune(proto.PaladinRune_RuneLegsAuraMastery) {
+		return
+	}
+
+	cd := core.Cooldown{
+		Timer:    paladin.NewTimer(),
+		Duration: time.Minute * 2,
+	}
+
+	aura := paladin.RegisterAura(core.Aura{
+		Label:    "Aura Mastery",
+		ActionID: core.ActionID{SpellID: 415756},
```

Comment:
This spell ID should be 407624, but you can also use `int32(proto.PaladinRune_RuneLegsAuraMastery)` since that has the same value

Reply:
will fix